python/tests/test_tree_stats.py

```diff
@@ -3350,6 +3331,14 @@ def test_bad_window_strings(self):
         with self.assertRaises(ValueError):
             ts.diversity([list(ts.samples())], mode="tree", windows="abc")
 
+    def test_bad_summary_function(self):
+        ts = self.get_tree_sequence()
+        W = np.ones((ts.num_samples, 3))
+        with self.assertRaises(ValueError):
+            _ = ts.general_stat(W, lambda x: x, 3, windows="sites")
```

Comment:
We don't need to assign this to ``_`` here I think as we're executing a function. I did this in the other weird example because I wasn't sure what would happen if you accessed an attribute but didn't assign it to any varible.